src/hooks/useMsgBusTracking.ts

```diff
@@ -0,0 +1,40 @@
+import { trackEvent, WALLET_EVENTS, MESSAGE_WALLET_EVENTS } from '@/services/analytics'
+import { SafeMsgEvent, safeMsgSubscribe } from '@/services/safe-messages/safeMsgEvents'
+import { TxEvent, txSubscribe } from '@/services/tx/txEvents'
+import { useEffect } from 'react'
+
+const txEvents = {
+  [TxEvent.SIGNED]: WALLET_EVENTS.OFF_CHAIN_SIGNATURE,
+  [TxEvent.PROCESSING]: WALLET_EVENTS.ON_CHAIN_INTERACTION,
+}
+
+const safeMsgEvents = {
+  [SafeMsgEvent.PROPOSE]: MESSAGE_WALLET_EVENTS.SIGN_MESSAGE,
+  [SafeMsgEvent.CONFIRM_PROPOSE]: MESSAGE_WALLET_EVENTS.CONFIRM_MESSAGE,
+}
+
+export const useMsgBusTracking = (): void => {
```

Comment:
That's a strange name. If it's tracking both transactions and messages, `MsgBus` is wrong.

Reply:
I've reverted the changes and created a separate one specifically for messages.